receiver/purefareceiver/config.go

```diff
@@ -46,6 +46,9 @@ type Config struct {
 
 	// Volumes represents the list of volumes to query
 	Volumes []internal.ScraperConfig `mapstructure:"volumes"`
+
+	// Env represents the list of environment to query
```

Comment:
If you are changing this PR for some reason, update this comment, as it's not a list of envs.